app/controllers/course/condition/levels_controller.rb

```diff
@@ -0,0 +1,43 @@
+class Course::Condition::LevelsController < Course::ConditionsController
+  load_resource instance_name: :level_condition, class: Course::Condition::Level.name,
```

Comment:
You mean just doing `load_resource :level_condition`? This doesn't work. I guess it's because it's difference from the model class.